src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx

```diff
@@ -162,14 +257,13 @@ export class CancerSummaryChart extends React.Component<CancerSummaryChartProps,
                             target: "data",
                             mutation: (props:any) => {
                                 if (props.datum.xKey in self.props.countsByGroup) {
-                                    self.tooltipModel = {
-                                        x:props.x + 20 - this.scrollPane.scrollLeft,
-                                        y:props.y - 18,
+                                    self.barPlotTooltipModel = {
+                                        ...props,
                                         groupName:props.datum.x,
                                         alterationData:self.props.countsByGroup[props.datum.xKey]
                                     };
                                 } else {
-                                    self.hideTooltip();
+                                    self.barPlotTooltipModel = null;
```

Comment:
why not just change this inside hideTooltip method?

Reply:
it's the same statement even in that method

Reply:
then why not just use it?   not important, but just trying to understand